service/feeder/live.go

```diff
@@ -0,0 +1,140 @@
+package feeder
+
+import (
+	"bytes"
+	"fmt"
+	"sync"
+
+	"github.com/fxamacker/cbor/v2"
+	"github.com/klauspost/compress/zstd"
+	"github.com/optakt/flow-dps/models/dps"
+	"github.com/rs/zerolog"
+	"go.nanomsg.org/mangos/v3"
+	"go.nanomsg.org/mangos/v3/protocol/sub"
+
+	// register transports
+	_ "go.nanomsg.org/mangos/v3/transport/all"
+
+	"github.com/onflow/flow-go/ledger"
+)
+
+const (
+	pubSubPort = 14532
+	reqPort    = 14533
+)
+
+type Live struct {
+	log zerolog.Logger
+
+	pubsub mangos.Socket
+	req    mangos.Socket
+
+	updatesMu    sync.Mutex
+	updates      []ledger.TrieUpdate
```

Comment:
I think we want to use the `deque.Deque` here. Internally, it uses a circular buffer that grows automatically, so it's way more efficient in terms of memory allocations and deallocations.